mteb/tasks/Image/Any2TextMultipleChoice/eng/CVBench.py

```diff
@@ -0,0 +1,266 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.Image.AbsTaskAny2TextMultipleChoice import (
+    AbsTaskAny2TextMultipleChoice,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+def transform_choices(example):
+    mapping = {"(A)": 0, "(B)": 1, "(C)": 2, "(D)": 3, "(E)": 4, "(F)": 5}
+    example["answer"] = mapping[example["answer"]]
+    return example
+
+
+class CVBenchCount(AbsTaskAny2TextMultipleChoice):
+    metadata = TaskMetadata(
+        name="CVBenchCount",
+        description="count the number of objects in the image.",
+        reference="https://arxiv.org/pdf/2406.16860",
+        dataset={
+            "path": "nyu-visionx/CV-Bench",
+            "revision": "22409a927ab5cf68e3655023d51694587455fc99",
+        },
+        type="Any2TextMutipleChoice",
+        category="it2t",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2024-01-01", "2024-12-31"),
+        domains=["Academic"],
+        task_subtypes=["Question answering"],
+        license="MIT",
+        annotations_creators="derived",
+        dialect=[],
+        modalities=["text", "image"],
+        sample_creation="found",
+        bibtex_citation="""@article{wu2024scimmir,
+  title={placeholder},
+  author={placeholder and others},
+  journal={arXiv preprint arXiv:2401.13478},
+  year={2024}
+}""",
+        descriptive_stats={
+            "n_samples": {"test": 788},
+            "avg_character_length": {
+                "test": {
+                    # to do
```

Comment:
Just wondering if this was a private note.

Reply:
yes. Still need to decide what are the useful metadata for this new AbsTask and image datasets in general. Think we can solve this as an issue at last!